### DISCUSSION TITLE
Admin Panel to View/Add/Delete users

### DISCUSSION THREAD

**Discussion by PaulMcF1987** [dani-garcia/vaultwarden#3429] [Q&A]
I installed Vaultwarden using the suggested `Docker Installation` method shown on the main page:

`docker pull vaultwarden/server:latest`
`docker run -d --name vaultwarden -v /vw-data/:/data/ -p 80:80 vaultwarden/server:latest`

My question is... I have no idea how to view/add/delete users... I have seen on other forums that there is an Admin page and in order to view this page you need to edit the .env file...

Using the install method above... where will I find the .env file to then edit it.

Any help would be much appreciated.

**Comment by BlackDex** [comment]
Check the wiki.
https://github.com/dani-garcia/vaultwarden/wiki/Enabling-admin-page

**Reply by PaulMcF1987** [reply]
Thank you for your reply. 

So if go into SSH and type

`docker run -d --name vaultwarden \`
`  -e ADMIN_TOKEN=some_random_token_as_per_above_explanation \`
`  -v /vw-data/:/data/ \`
`  -p 80:80 \`
`  vaultwarden/server:latest`

this shoul enable the admin panel without compromising my current install?

**Reply by BlackDex** [reply]
I would suggest to read this wiki reading that.
https://github.com/dani-garcia/vaultwarden/wiki/Updating-the-vaultwarden-image

Steps are the same for updating an image of modifying env variables.

**Reply by PaulMcF1987** [reply]
So running through that but making sure that i add `-e ADMIN_TOKEN=some_random_token_as_per_above_explanation \` should allow me to access the admin panel?

One more question on this...
Usually I use docker compose and use a `YAML` and `.env` file but I could not find one for Vaultwarden.
How do i know if I `bind mounted the volume for persistent data`
Would I be better off using the steps from the second section to be safe?

**Reply by BlackDex** [reply]
Again, check the wiki.
https://github.com/dani-garcia/vaultwarden/wiki/Using-Docker-Compose

All env variables can be checked here 
https://github.com/dani-garcia/vaultwarden/blob/0b28ab3be101b58cf27b43da14bcf893fa37c1c8/.env.template

Note that those values are not to be mistaken for defaults.

**Reply by PaulMcF1987** [reply]
@BlackDex I sorted it. I found an area within Plesk where I could add a .env variable

thank you for your help with this!
